crd/apis/network/v1/gkenetworkparamset_types.go

```diff
@@ -64,14 +64,77 @@ type NetworkRanges struct {
 	CIDRBlocks []string `json:"cidrBlocks"`
 }
 
+// GKENetworkParamSetConditionType is the type for status conditions on
+// a GKENetworkParamSet. This type should be used with the
+// GKENetworkParamSetStatus.Conditions field.
+type GKENetworkParamSetConditionType string
+
+const (
+	// GKENetworkParamSetStatusReady is the condition type that holds
+	// if the GKENetworkParamSet object is validated
+	GKENetworkParamSetStatusReady GKENetworkParamSetConditionType = "Ready"
+)
+
+// GKENetworkParamSetReason defines the set of reasons that explain why a
+// particular GKENetworkParamSet condition type has been raised.
+type GKENetworkParamSetReason string
```

Comment:
How about `GKENetworkParamSetReason` -> `GKENetworkParamSetConditionReason`, since you are calling `GKENetworkParamSetConditionType` above.

Reply:
youre right, good catch. done.